test/regress/regress0/bv/eager-inc-cryptominisat_force_logic.smt2

```diff
@@ -0,0 +1,26 @@
+; REQUIRES: cryptominisat
+; COMMAND-LINE: --incremental --bv-sat-solver=cryptominisat --bitblast=eager --force-logic="QF_BV"
```

Comment:
Can you maybe use a test that does not require extra dependencies (so that Travis also checks it)?

Reply:
Good point. We had a similar issue when producing models with eager bitblasting because [the checks](https://github.com/CVC4/CVC4/blob/master/src/smt/smt_engine.cpp#L1185-L1192) happened before resetting to the forced logic. I've changed the regression s.t. it triggers that case on master.